sky/global_user_state.py

```diff
@@ -264,6 +265,34 @@ class ClusterEventType(enum.Enum):
     sqlalchemy.Column('updated_at', sqlalchemy.Integer),
 )
 
+cluster_failures_table = sqlalchemy.Table(
+    'cluster_failures',
+    Base.metadata,
+    sqlalchemy.Column('id',
+                      sqlalchemy.Integer,
+                      primary_key=True,
+                      autoincrement=True),
+    sqlalchemy.Column('cluster_hash', sqlalchemy.Text, index=True),
+    # failure_mode is a machine readable code for the failure mode.
+    # For example, "GPU_APPLICATION_ERROR_XID_CODE_31" or
+    # "GPU_HARDWARE_FAILURE_XID_CODE_32".
+    sqlalchemy.Column('failure_mode', sqlalchemy.Text, index=True),
```

Comment:
We have both `failure_mode` and `failure_code` used in the codebase. Could we consolidate the naming for them?